rbac/app/config.py

```diff
@@ -35,6 +35,8 @@
     "DB_NAME": "rbac",
     "CHATBOT_HOST": "chatbot",
     "CHATBOT_PORT": "5005",
+    "ADAPI_HOST": "adapi.corporate.t-mobile.com",
```

Comment:
We should not disclose our internal resources as a default value. Probably just leave the default out? Seems this is not required to have our code to be functioning.

Reply:
Removed. 👾